### PR TITLE
DOCS Adjust owns upgrading example

### LINKED ISSUE #7598: Documentation of owns
The documentation of the "New Ownership API" has a flaw.  If you search the page for "$owns" there is one occurene in the code.  It is only occurence of "owns" also.  Unless a huge screen and a small font it is impossible to tell you are in the section documenting "$owns".  So put an example directly below the h3 

Like

`$owns = <array of class names>`

but in the style you like.

Would hjhave saved at least one question on slack!

### PR DESCRIPTION
**Pull request by tractorcow**
Fixes #7598

### CODE REVIEW

**Comment by tractorcow** [comment]
Tests fixed with https://github.com/silverstripe/silverstripe-cms/pull/2028

File: docs/en/04_Changelogs/4.0.0.md [hunk 1]
```diff
@@ -1350,18 +1350,26 @@ These methods are deprecated:
...
-components could be implicitly exposed on the frontend, which may not align with the intent of the
-content author.
+developers to declare dependencies between versioned objects.
 
-For instance, on a products page which has a list of products, the products should not be published unless the products page is, too. The ownership API solves this by allowing you to declare
- a two-way relationship between objects, typically, but not necessarily, linked by a database relationship
- (`has_many`, `many_many`, etc.).
+```php
+private static $owns = [
+    'Banners', // This page depends on banners, declared as a separate 'has_mane'
```
Comment (christopherdarling) [inline docs/en/04_Changelogs/4.0.0.md]: typo on 'has_many'

**Comment by dhensby** [comment]
fixed typo when merged by hand

File: docs/en/04_Changelogs/4.0.0.md (hunk 1 above)
Comment (tractorcow) [reply docs/en/04_Changelogs/4.0.0.md]: It only works for lions obviously. :)

**Pull request merged**